### DISCUSSION TITLE
GitHub Integration With Unturned.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3652] [Ideas]
Just an idea I figured was worth mentioning for the future,

As of writing this, Unturned has no multiplayer map editing. This is unfortunate but understandable because of the work that would be required to get it even partially working. An interesting idea to at least improve the situation would be to add GitHub integration (in the far future of course). I've seen a couple of map makers that upload their maps to GitHub in order to collaborate more easily.

**Comment by DrDallas1** [comment]
I think it would take a lot more time to make this work. And it wouldn't be much different from what can already be done. (Github is incredible)
